### ISSUE TITLE
Can we have separate rules for exception builtins?

### ISSUE THREAD

**Issue by cielavenir** [gforcada/flake8-builtins#126]
The source code

```
from requests.exceptions import ConnectionError
```

Raises `A004 import statement "ConnectionError" is shadowing a Python builtin`.

I understand that ConnectionError is a builtin, which is a subclass of OSError. However, `ConnectionError` is quite widely used term.

The more issue is, when we apply flake8-builtins in CI, we cannot make a new connection module which provides ConnectionError.

Can we have separate rules for such exception builtins?

/cc @felixvd @hemangandhi @ntohge

**Comment by gforcada** [comment]
Hi @cielavenir! thanks for using `flake8-builtins` and taking the time to make this report!

You can use the `--builtins-ignorelist ConnectionError` command line option, or configuration option to ignore builtins that you consider safe to override 👍🏾 

Another option would be to change the code to:

```python
from requests import exceptions

...

try:
  ...
except exception.ConnectionError
    ...
```

I hope this helps! 🍀
